src/shims/ApiCompat.proj

```diff
@@ -40,6 +41,16 @@
     </Exec>
 
     <Exec Command="$(ApiCompatCmd) &quot;$(NetStandardRefPath)&quot; @&quot;$(ApiCompatResponseFile)&quot; &gt; $(ApiCompatNSBaselineFile)"
+          Condition="'$(TargetGroup)' != 'netfx'"
```

Comment:
Right, I built directly in the project with /p:targetgroup=netfx and it didn't complain. I'll remove then then.